package.json

```diff
@@ -17,7 +17,7 @@
     "examples/sam"
   ],
   "scripts": {
-    "init-environment": "husky install",
+    "init-environment": "husky",
```

Comment:
Let's remove this entirely & move `husky` in the `setup-local` since it's now only a single keyword/command.